verticapy/core/vdataframe/_aggregate.py

```diff
@@ -1425,6 +1934,57 @@ def median(
         -------
         TableSample
             result.
+
+        Examples
+        --------
+        For this example, we will use the following dataset:
+
+        .. code-block:: python
+
+            import verticapy as vp
+
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+        
+        Now, let's calculate the median for specific columns.
+
+        .. code-block:: python
+
+            data.median(
+                columns = ["x", "y", "z"],,
+                approx = True,
+            )
+
+        .. ipython:: python
+            :suppress:
+
+            import verticapy as vp
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+            result = data.median(
+                columns = ["x", "y", "z"],,
```

Comment:
Here as well. You can remove the last extra comma. 

columns = ["x", "y", "z"],**,**